internal/cephfs/util.go

```diff
@@ -32,6 +32,15 @@ import (
 	"k8s.io/klog"
 )
 
+// enum defining logging levels.
```

Comment:
This file is not a good location to place this in. Please consider moving it to the shared `util` package. You could even have helper functions like `util.Debug(...)` that calls the appropriate `klog.V()` functions.

Reply:
In order to have the right filename in the log, you probably need to call `klog.V(x).InfoDepth(...)` with a `depth` of `1`. Otherwise you'd see the filename where your log helper function is implemented.

Reply:
You can also have these kind of changes in their own PR. That makes reviewing and updating a lot easier.

Reply:
Sure, I will bring up a seperate PR for this one.